dh/service.go

```diff
@@ -10,7 +10,38 @@ const (
 	Timeout          = 5 * time.Second
 )
 
-func Connect(url string) (client *Client, err *Error) {
+func ConnectWithToken(url, accessToken, refreshToken string) (client *Client, err *Error) {
```

Comment:
Introduce struct with login, password, refreshToken, accessToken and single Connect method